data/presets/presets/craft/metal_working.json

```diff
@@ -7,5 +7,5 @@
     "tags": {
         "craft": "metal_construction"
     },
-    "name": "Metal Construction"
+    "name": "Metalworking"
```

Comment:
Let’s move this file back to where it was. I don’t think anything depends on the file names being consistent with the tags, but it’ll be easier to find the preset at the old name.